packages/features/src/index.ts

```diff
@@ -1,6 +1,7 @@
 import { stringifyBbox } from './bbox';
 import { stringifyDatetime, IDateRange  } from './datetime';
 import { stringifyProperties } from './properties';
+import { stringifySortBy, TSortBy } from './sortby';
```

Comment:
I think you should also export the `TSortBy` type and the `ISortByItem` interface for the user.

Reply:
Yep, seems like a good idea.